pydm/main_window.py

```diff
@@ -304,7 +312,12 @@ def open_editor_generic(fname):
     @pyqtSlot(bool)
     def open_file_action(self, checked):
         modifiers = QApplication.keyboardModifiers()
-        filename = QFileDialog.getOpenFileName(self, 'Open File...', os.path.dirname(self.current_file()), 'PyDM Display Files (*.ui *.py)')
+        try:
+            curr_file = self.current_file()
+        except IndexError:
+            curr_file = os.getcwd()
+
+        filename = QFileDialog.getOpenFileName(self, 'Open File...', os.path.dirname(curr_file), 'PyDM Display Files (*.ui *.py)')
```

Comment:
I think this may go one level further up the tree than you think if there is no `current_file()`
```python
curr_file = os.getcwd()   # gets working directory
os.path.dirname(curr_file)  # gets directory of current working directory 
```

Reply:
You are right!
Addressed on the latest commit.